js/packages/web/src/views/auctionCreate/priceAuction.tsx

```diff
@@ -1,90 +1,173 @@
-import { Button, Input, Space } from 'antd';
-import React from 'react';
+import { Button, InputNumber, Space, Form } from 'antd';
+import { useForm } from 'antd/lib/form/Form';
+import React, { useState } from 'react';
 import { AuctionCategory, AuctionState } from '.';
 
 export const PriceAuction = (props: {
   attributes: AuctionState;
   setAttributes: (attr: AuctionState) => void;
   confirm: () => void;
 }) => {
+  type InputState = number | null;
+  const [price, setPrice] = useState<InputState>(null);
+  const [tickSize, setTickSize] = useState<InputState>(null);
```

Comment:
Can we drop the the price and tickSize as local state? Its tracked in a attributes yes?

Reply:
Yeah that's cleaner.  Resubmitted, dropping the local state